spec/models/contact_spec.rb

```diff
@@ -1091,20 +1091,20 @@ class Xxx < LocalUniqueAttribute; end
 
         HistoryEntry.create(attribute: 'local_status_for_martinez',
                             old_value: '',
-                            changed_at: DateTime.civil(2012,11,21,20,34,39).to_time,
+                            changed_at: "2012-11-21 20:34:39".to_time(:utc),
                             historiable_type: 'Contact',
                             historiable_id: @contact._id
         )
         HistoryEntry.create(attribute: 'local_status_for_martinez',
                             old_value: :prospect,
-                            changed_at: DateTime.civil(2012,11,21,20,35,50).to_time,
+                            changed_at: "2012-11-21 20:35:50".to_time(:utc),
                             historiable_type: 'Contact',
                             historiable_id: @contact._id
         )
       end
-      example { contacts_with_value_at('student',Date.civil(2012,11,20)).should_not include @contact}
-      example { contacts_with_value_at('student',Date.civil(2012,11,22)).should include @contact}
-      example { contacts_with_value_at('prospect',DateTime.civil(2012,11,21,20,34,41).to_time).should include @contact }
+      example { contacts_with_value_at('student', "2012-11-20").should_not include @contact}
```

Comment:
Line is too long. [92/80]<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.